server/fishtest/templates/tests_view.mak

```diff
@@ -4,6 +4,11 @@
   import json
 %>
 
+<%
```

Comment:
Independent from template arguments, move in this tag.
```
<%!
```

Reply:
Ooops, it seems that I was wrong, please revert this change, it seems that in that way the names become const. 

```yaml
May 27 15:45:29 dfts-0 pserve[27882]:   File "/home/usr00/fishtest/server/fishtest/templates/tests_view.mak", line 80, in render_body
May 27 15:45:29 dfts-0 pserve[27882]:     <% args_row_count += 1 %>
May 27 15:45:29 dfts-0 pserve[27882]: UnboundLocalError: cannot access local variable 'args_row_count' where it is not associated with a value
```